assets/core.styl

```diff
@@ -130,3 +130,4 @@ resets(arr)
   content: attr(data-placeholder)
   font-style: italic
   position: absolute
+  pointer-events:none
```

Comment:
Can you alphabetize the rule and add a space after the colon to keep with existing styles?